tracpro/templates/workflow_tracker_steps.html

```diff
@@ -0,0 +1,40 @@
+<div class="row workflow-tracker">
+    <a href="{% url 'groups.region_list' %}">
+        <div class="col-md-2 col-sm-1 hidden-xs {% if step == 1 %}circle-selected{% else %}circle{% endif %}">
+            1
+            <div class="circle-label">
+                Select Panels
+            </div>
+        </div>
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'groups.group_list' %}">
+        <div class="col-md-2 col-sm-1 hidden-xs {% if step == 2 %}circle-selected{% else %}circle{% endif %}">
+            2
+            <div class="circle-label">
+                Select Cohorts
+            </div>
+        </div>
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'polls.poll_list' %}">
+        <div class="col-md-2 col-sm-1 hidden-xs {% if step == 3 %}circle-selected{% else %}circle{% endif %}">
+            3
+            <div class="circle-label">
+                Select Flows
+            </div>
+        </div>
+    </a>
+    <div class="col-md-2 hidden-sm hidden-xs workflow-spacer"></div>
+
+    <a href="{% url 'orgs_ext.org_fetchruns' %}">
```

Comment:
We should only display the 4th circle if the user is authorized to do fetchruns.

(```  {% if request.user.is_superuser or request.user.is_staff %}``` is what we've used elsewhere.)